_includes/header.html

```diff
@@ -4,8 +4,8 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="shortcut icon" href="/assets/img/favicon.ico" type="image/x-icon">
-  <link rel="stylesheet" href="/assets/css/style.css">
+  <link rel="shortcut icon" href="{{ "/assets/img/favicon.ico" | relative_url }}" type="image/x-icon">
+  <link rel="stylesheet" type="text/css" href="{{ "/assets/css/style.css" | relative_url }}">
```

Comment:
No `type` stuff for HTML5

Reply:
Hi @nschonni - yes they are - I tried to find most/all of them and update.

I am still hitting some errors in TravisCI
```
- ./_site/fedramp-confirmation/index.html
  *  External link https://government.github.com/fedramp-confirmation/ failed: 404 No error
- ./_site/fedramp/index.html
  *  External link https://government.github.com/fedramp/ failed: 404 No error
```

and I am not sure how to remedy as those links look like they should be ok.

I kind of suspect it ties back to these lines in config.yml
```
url: https://government.github.com
relative_permalinks: false
```
but not really sure.  I am not a Jekyll expert by any means.  The resources site, which functions similarly, does not have those in the config.

cc: @jbjonesjr @sophshep

Reply:
@mike421453 yeah, so travis makes sure all links on the site are valid.  Since those two links (fedramp-confirmation and /fedramp) return `404` right now, travis is failing the build.  You can ignore those errors, especially if your PR is going to create those pages.